topics/RDMA.md

```diff
@@ -0,0 +1,198 @@
+---
+title: "RDMA support"
+linkTitle: "RDMA support"
+description: Valkey Over RDMA support
+---
+
+Valkey supports the Remote Direct Memory Access (RDMA) connection type via a
+Valkey module that can be dynamically loaded on demand.
+
+## Getting Started
+
+[RDMA](https://en.wikipedia.org/wiki/Remote_direct_memory_access)
+enables direct data exchange between networked computers' main memory,
+bypassing processors and operating systems.
+
+As a result, RDMA offers better performance compared to TCP/IP. Test results indicate that
+Valkey Over RDMA achieves approximately 2 times higher QPS and lower latency.
+
+Please note that Valkey Over RDMA is currently supported only on Linux.
+
+## Running manually
```

Comment:
Is there a non-manual way to run it?  If so, how?

Reply:
Sure, add valkey.conf sample in next version.

Reply:
Perhaps "Running Manually" should be "Specifying RDMA on the Valkey command line".